app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -0,0 +1,545 @@
+                </Pressable>
+            </ScrollView>
+        );
+    }
+
+    const HandIcon = myParticipant.raisedHand ? UnraisedHandIcon : RaisedHandIcon;
+
+    return (
+        <SafeAreaView style={style.wrapper}>
+            <View style={style.container}>
+                <View
+                    style={[style.header, isLandscape && style.headerLandscape, !showControls && style.headerLandscapeNoControls]}
+                >
+                    <CallDuration
+                        style={style.time}
+                        value={currentCall.startTime}
+                        updateIntervalInSeconds={1}
+                    />
+                    <Pressable onPress={() => popTopScreen()}>
+                        <CompassIcon
+                            name='arrow-collapse'
+                            size={24}
+                            style={style.collapseIcon}
+                        />
+                    </Pressable>
+                </View>
+                {usersList}
+                {screenShareView}
+                <View
+                    style={[style.buttons, isLandscape && style.buttonsLandscape, !showControls && style.buttonsLandscapeNoControls]}
+                >
+                    {!isLandscape &&
+                        <Pressable
+                            testID='mute-unmute'
+                            style={[style.mute, myParticipant.muted && style.muteMuted]}
+                            onPress={muteUnmuteHandler}
+                        >
+                            <CompassIcon
+                                name={myParticipant.muted ? 'microphone-off' : 'microphone'}
+                                size={24}
+                                style={style.muteIcon}
+                            />
+                            {myParticipant.muted &&
+                                <Text style={style.buttonText}>{'Unmute'}</Text>}
+                            {!myParticipant.muted &&
+                                <Text style={style.buttonText}>{'Mute'}</Text>}
+                        </Pressable>}
+                    <View style={style.otherButtons}>
+                        <Pressable
+                            testID='leave'
+                            style={style.button}
+                            onPress={leaveCallHandler}
+                        >
+                            <CompassIcon
+                                name='phone-hangup'
+                                size={24}
+                                style={{...style.buttonIcon, ...style.hangUpIcon}}
+                            />
+                            <Text style={style.buttonText}>{'Leave'}</Text>
+                        </Pressable>
+                        <Pressable
+                            testID={'toggle-speakerphone'}
+                            style={style.button}
+                            onPress={() => setSpeakerphoneOn(!currentCall?.speakerphoneOn)}
```

Comment:
definitely avoid inline functions to minimize the renders as I can see this component re-rendering a hell of a lot as new participants join/leave the call.